src/test/java/org/opentripplanner/transit/speed_test/SpeedTest.java

```diff
@@ -117,6 +119,7 @@ public SpeedTest(
         TestServerContext.createVehicleRentalService(),
         config.flexConfig,
         List.of(),
+        new DefaultStopConsolidationService(new DefaultStopConsolidationRepository(), transitModel),
```

Comment:
I've made the service and repo nullable here: 953216df9be78625520a482a4397928177f54f61

Reply:
So, design is about tradeoffs - everyone agree that nulls are a potensial hasard. But, here there are a few much bigger problems risks. I do not trust Sandbox code! I do not want to look inside the to make sure it is safe. Also, services or plugins witch are by default off can safely be left as null - because if the default configuration run then it is null safe - so there is no real danger here. 

The alternative is to make a extention point, in this case an interface in the main code that is not specific to the `DefaultStopConsolidationService`, but passes in enough information to it to do its job. Then you can make a NOOP implementation of the interface in the main code. This way you get rid of the null. But, ever part of the main code that does not directly interact - and it should not interact with Sandbox code - should not be coserned with how this constructed.